### ISSUE TITLE
Minor issue: thin lines remain with `combmatrix.panel.line.size = 0`

### ISSUE THREAD

**Issue by karldw** [const-ae/ggupset#12]
Hi @const-ae, 

I noticed that if I set `theme_combmatrix(combmatrix.panel.line.size = 0)` and save the plot as a PDF, I still get thin lines connecting the sets. This is super minor, I just thought I should note it in case others are confused about their PDF output.

Example:
```r
library(ggplot2)
library(ggupset)
library(dplyr)
plt <- tidy_movies %>%
  distinct(title, year, length, .keep_all=TRUE) %>%
  ggplot(aes(x=Genres)) +
    geom_bar() +
    scale_x_upset() +
    theme_combmatrix(combmatrix.panel.line.size = 0)
ggsave("test_plot.pdf", plot=plt)

```
[test_plot.pdf](https://github.com/const-ae/ggupset/files/4495553/test_plot.pdf)

**Comment by const-ae** [comment]
Hi Karl, 

thanks for reporting the issue. I agree that this is annoying. 
However, I am swamped this week with other deadlines. The fix is probably straight forward: one would need to check if `theme$combmatrix.panel.line.size != 0` and then add the vertical lines conditional on that (https://github.com/const-ae/ggupset/blob/master/R/axis_combmatrix.R#L259). Would you be interested in giving it a try and write a pull request?

Best, Constantin

**Comment by karldw** [comment]
I'd be happy to write a PR! It will be a couple of weeks, but I'll add it to my list.

**Comment by const-ae** [comment]
Great :) If you have any questions during the process, just ask away :)

**Comment by const-ae** [comment]
I have submitted the fixed version 0.3.0 to CRAN and should become available in the next days.
